### ISSUE TITLE
welcome message

### ISSUE THREAD

**Issue by mokkin** [nextcloud/richdocuments#1034]
**Is your feature request related to a problem? Please describe.**
Everytime opening a document Collabora welcome message appears.

**Describe the solution you'd like**
Only once or never show the welcome message.

**Describe alternatives you've considered**
Make it configurable.

**Comment by timar** [comment]
Oh wow, if the welcome message appears continually that's a serious
bug. It should show up just once per user / per version. Can you confirm
you get it more than once in each browser ?

	Checkout loolwsd.xml - for the setting to disable.

	But - we badly want to debug this to help other users.

	Can you tell me what browser & OS you are using ?

	Can you look at the F12/Javascript console to see if there is some
error there that might be helpful ?

	Can you give some information on your web server, version & setup.

	Help much appreciated,

	Thanks !

**Comment by schorsch2** [comment]
I can confirm, the note appears every time.
richdocuments 3.5.3
CODE  4.2.4
Nextcloud 19.0.0

**Comment by rgl1234** [comment]
How can this welcome-message be disable?

**Comment by Lurkars** [comment]
On selfthosted instance, you can disable welcome screen in loolwsd.xml
`    
<welcome>
      <enable type="bool" desc="Controls whether the welcome screen should be shown to the users on new install and updates." default="false">false</enable>
      <enable_button type="bool" desc="Controls whether the welcome screen should have an explanatory button instead of an X button to close the dialog." default="false">false</enable_button>
      <path desc="Path to 'welcome-$lang.html' files served on first start or when the version changes. When empty, defaults to the Release notes." type="path" relative="true" default="loleaflet/welcome>
    </welcome>
`

**Comment by nsrafk** [comment]
I see the same issue. Running dockerized version so directly editing loolwsd isn’t that good of a deal. Can we pass a extra_params for it maybe?

**Comment by thebearon** [comment]
> I see the same issue. Running dockerized version so directly editing loolwsd isn’t that good of a deal. Can we pass a extra_params for it maybe?

You can copy files from/to the container, thus you can copy `/etc/loolwsd/loolwsd.xml` out, edit it, copy it back in, and restart the container.
https://docs.docker.com/engine/reference/commandline/cp/

However, it'd be great to know more details about why the welcome message keeps showing. It is supposed to store a flag (actually, the version) in the browser's local storage, perhaps that fails for some reason... but why?

**Comment by nijel** [comment]
> I see the same issue. Running dockerized version so directly editing loolwsd isn’t that good of a deal. Can we pass a extra_params for it maybe?

Following does the trick for me:

```yaml
      extra_params: -o:welcome.enable=false
```

**Comment by juliushaertl** [comment]
Closing as this is out of scope of the Nextcloud app.
